src/main/java/com/erigitic/commands/JobCommand.java

```diff
@@ -221,8 +223,37 @@ public CommandResult execute(CommandSource src, CommandContext args) throws Comm
                     TEJobSet jobSet = optSet.get();
                     Currency defaultCurrency = totalEconomy.getDefaultCurrency();
 
-                    for (TEActionReward actionReward : jobSet.getActionRewards()) {
-                        lines.add(Text.of(TextColors.GOLD, "[", jobManager.titleize(actionReward.getAction()), "] ", TextColors.GRAY, actionReward.getTargetID(), TextColors.GOLD, " (", actionReward.getExpReward(), " EXP) (", defaultCurrency.format(actionReward.getMoneyReward()), ")"));
+                    for (TEAction action : jobSet.getActions()) {
+                        EconomyService service = Sponge.getServiceManager().provideUnchecked(EconomyService.class);
+                        Optional<Currency> c = Optional.empty();
+                        Text listText;
+
+                        if (action.isIDTraited()) {
+                            // MC does not support '\t'
+                            String tab = new String(new char[action.getAction().length() + 2]).replace("\0", " ");
+                            List<Text> texts = new ArrayList<>(action.getRewards().size());
+
+                            action.getRewards().forEach((k, v) -> {
+                                Text t1 = Text.of("");
+
+                                if (action.isGrowing()) {
+                                    t1 = Text.of(",growing=1");
+                                }
+
+                                Text t2 = (Text.of('\n', tab,  TextColors.GRAY, "{", action.getIDTrait(), '=', k, t1, "} ", TextColors.GOLD, format_reward(v)));
+                                texts.add(t2);
+                            });
+
+                            listText = Text.join(texts.toArray(new Text[texts.size()]));
+                        } else {
+
+                            listText = format_reward(action.getReward().get());
+
+                            if (action.isGrowing()) {
+                                listText = Text.of(TextColors.GRAY, "{growing=1} ", listText);
```

Comment:
The thought is to provide an indication for whether growth is taken into account on this action.

Reply:
My concern is that the user will see this and have no idea what it means. As long as it's being properly handled from the backend, I think it would be best to remove it from the frontend in order to avoid creating any confusion for the player.

Reply:
There're the users on the one hand but please don't forget supporters/moderators and admins who most times need specific information in order to help normal users or configure settings.

Reply:
Let's add an optional parameter to the job info command that'll show the extra information such as `growing=1`. Something like /job info e(xtended). Make it an optional thing, as most users won't have a need for this information all the time.

Reply:
hmm... yeah I guess that'll do.